src/pal/src/thread/process.cpp

```diff
@@ -825,28 +822,6 @@ CorUnix::InternalCreateProcess(
             palError = ERROR_FILE_NOT_FOUND;
             goto InternalCreateProcessExit;
 
-        case FILE_PE: /* PE/COFF file */
```

Comment:
Should we still explicitly handle `FILE_PE` since it seems we are handling ELF below?